gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -1215,6 +1273,81 @@ def get_network_function_device_config_info(self, device_data,
                                          if consumer_cidr
                                          else [provider_cidr]),
                         'destination_cidr': consumer_cidr,
+                        'provider_mac': provider_mac,
+                        'gateway_ip': consumer_gateway_ip,
+                        'provider_interface_index': 2
+                    }
+                }
+            ]
+        }
+
+
+
+    @_set_network_handler
+    def get_create_network_function_device_config_info(self, device_data,
+                                                network_handler=None):
+        """ Get the configuration information for NFD
```

Comment:
RPM, Please check, "get_network_function_device_config_info" is getting everything doing REST APIs, "get_create_network_function_device_config_info" is getting everything from "device_data" - optimized.